### PR TITLE
adds osclass chart

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by prydonius** [comment]
@k8s-bot ok to test

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

File: stable/osclass/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: mariadb
+  version: 0.5.4
```
Comment (viglesiasce) [inline stable/osclass/requirements.yaml]: Please update to 0.5.6 which has the fix for K8s 1.5.1

**Review by viglesiasce — CHANGES_REQUESTED** [review]

**Comment by sameersbn** [comment]
- updated mariadb chart version to 0.5.6
- `imagePullPolicy` explicitly set to `IfNotPresent`

**Comment by viglesiasce** [comment]
@k8s-bot e2e test this

**Review by viglesiasce — APPROVED** [review]

**Pull request merged**